Cleanup OpenCensus warnings in bigtable example.

We are getting the following warnings when compiling with Bazel (which enables OpenCensus):

```
google/cloud/bigtable/examples/opencensus/bigtable_opencensus.cc:73:54: warning: 'Register' is deprecated: Register() without StackdriverOptions is deprecated and will be removed on or after 2019-03-20 [-Wdeprecated-declarations]
  opencensus::exporters::stats::StackdriverExporter::Register(
                                                     ^
external/io_opencensus_cpp/opencensus/exporters/stats/stackdriver/stackdriver_exporter.h:49:3: note: 'Register' has been explicitly marked deprecated here
  ABSL_DEPRECATED(
  ^
external/com_google_absl/absl/base/macros.h:148:49: note: expanded from macro 'ABSL_DEPRECATED'
#define ABSL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                ^
google/cloud/bigtable/examples/opencensus/bigtable_opencensus.cc:75:54: warning: 'Register' is deprecated: Register() without StackdriverOptions is deprecated and will be removed on or after 2019-03-20 [-Wdeprecated-declarations]
  opencensus::exporters::trace::StackdriverExporter::Register(project_id);
                                                     ^
external/io_opencensus_cpp/opencensus/exporters/trace/stackdriver/stackdriver_exporter.h:42:3: note: 'Register' has been explicitly marked deprecated here
  ABSL_DEPRECATED(
  ^
external/com_google_absl/absl/base/macros.h:148:49: note: expanded from macro 'ABSL_DEPRECATED'
#define ABSL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                ^
2 warnings generated.
```

--- a/google/cloud/bigtable/examples/opencensus/bigtable_opencensus.cc
+++ b/google/cloud/bigtable/examples/opencensus/bigtable_opencensus.cc
@@ -70,9 +70,15 @@ int main(int argc, char* argv[]) try {
   // For more details, see the documentation
   //   https://github.com/census-instrumentation/opencensus-cpp/tree/master/opencensus/exporters/stats/stackdriver#opencensus-stackdriver-stats-exporter
 
-  opencensus::exporters::stats::StackdriverExporter::Register(
-      project_id, "bigtable-opencensus-0@unspecified-host");
-  opencensus::exporters::trace::StackdriverExporter::Register(project_id);
+  opencensus::exporters::stats::StackdriverOptions stats_opts;
+  stats_opts.project_id = project_id;
+  stats_opts.opencensus_task = "bigtable-opencensus-0@unspecified-host";
+
+  opencensus::exporters::trace::StackdriverOptions trace_opts;
+  trace_opts.project_id = project_id;
+
+  opencensus::exporters::stats::StackdriverExporter::Register(stats_opts);
+  opencensus::exporters::trace::StackdriverExporter::Register(trace_opts);
 
   // Connect to the Cloud Bigtable Admin API.
   //! [connect admin]

